hardening/host-os/ansible/test.py

```diff
@@ -54,11 +54,13 @@
     # old RHEL-7 oscap mixes errors into --progress rule names without a newline
     verbose = ['--verbose', 'INFO'] if versions.oscap >= 1.3 else []
     redir = {'stderr': subprocess.STDOUT} if versions.oscap >= 1.3 else {}
+    # RHEL-7 HTML report doesn't contain OVAL findings by default
+    oval_results = '' if versions.oscap >= 1.3 else ['--results', 'results.xml', '--oval-results']
```

Comment:
You used only half of my correction. :smile:, but `*` expansion of an empty string seems to be somehow magically working, though using `[]` would keep the same data type.

Reply:
D'oh. Fixed